figures/metrics.py

```diff
@@ -232,6 +236,50 @@ def course_progress(course_enrollment):
 # TODO: move these to `figures.metrics.site` module
 #
 
+def get_site_mau_history_metrics(site, months_back):
+    """Quick adaptation of `get_monthly_history_metric` for site MAU
+    """
+    history = []
+
+    # We will not have the current month's data because metrics are calculated
+    # after the month is over
+    for rec in SiteMonthlyMetrics.objects.order_by('-month_for')[:months_back]:
+        period = '{year}/{month}'.format(year=rec.month_for.year,
+                                         month=str(rec.month_for.month).zfill(2))
+        history.append(dict(period=period, value=rec.active_user_count))
+
+    # Hack to set current month data in the history list
+    current_month_active = get_site_mau_current_month(site)
+    if history:
+        # reverse the list because it is currently in reverser chronological order
+        history.reverse()
+
+    current_month = datetime.datetime.utcnow().date()
+    period = '{year}/{month}'.format(year=current_month.year,
+                                     month=str(current_month.month).zfill(2))
+    history.append(dict(period=period, value=current_month_active))
+    return dict(current_month=current_month_active, history=history)
+
+
+def get_site_mau_current_month(site):
+    """
+    WIP for performance optimizsation
```

Comment:
Is this comment still valid?

Reply:
@OmarIthawi I'll update the comment to summarize status and do a TODO to explain in more detail for follow up